homeassistant/components/system_bridge/coordinator.py

```diff
@@ -75,6 +75,17 @@ def __init__(
             hass, LOGGER, name=DOMAIN, update_interval=timedelta(seconds=30)
         )
 
+    @property
+    def is_ready(self) -> bool:
+        """Return if the data is ready."""
+        if self.data is None:
+            return False
+        for module in MODULES:
+            if getattr(self.data, module) is None:
+                self.logger.debug("%s - Module %s is None", self.title, module)
```

Comment:
Please don't do side effects in properties. If you want to do that, please make it a method instead.